### PR TITLE
Add an example with Workbox CLI

### PR DESCRIPTION
**Pull request by fheslouin**
## Description

I spent lot of time trying to configure gatsby-plugin-offline to get a full offline website experience. This add a working example with Workbox CLI to get a quickly working configuration.

### CODE REVIEW

**Comment by laurieontech** [comment]
Thanks for the PR! Adding this to the list for technical verification.

File: packages/gatsby-plugin-offline/README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ plugins: [
 
 In version 3, Workbox is also upgraded to version 4 so you may need to update your `workboxConfig` if any of those changes apply to you. Please see the [docs on Google Developers](https://developers.google.com/web/tools/workbox/guides/migrations/migrate-from-v3) for more information.
 
-## Overriding Workbox configuration
+## Overriding Workbox (v4.3.1) configuration
```
Comment (laurieontech) [inline packages/gatsby-plugin-offline/README.md]:
```suggestion
## Overriding Workbox configuration
```
I'd rather we note the specific version, if necessary, in the text.

File: packages/gatsby-plugin-offline/README.md [hunk 2]
```diff
@@ -166,6 +166,28 @@ const options = {
 }
 ```
 
+You can also use [Workbox CLI](https://developers.google.com/web/tools/workbox/modules/workbox-cli) to generate this file automatically.
```
Comment (laurieontech) [inline packages/gatsby-plugin-offline/README.md]: If this is going to list the instructions for using workbox CLI perhaps it should include the npm install command from that link as well?

File: packages/gatsby-plugin-offline/README.md [hunk 3]
```diff
@@ -166,6 +166,28 @@ const options = {
 }
 ```
 
+You can also use [Workbox CLI](https://developers.google.com/web/tools/workbox/modules/workbox-cli) to generate this file automatically. 
+
+```
+workbox wizard
```
Comment (laurieontech) [inline packages/gatsby-plugin-offline/README.md]: It should be explicit that `workbox wizard` is a command the user needs to run.

File: packages/gatsby-plugin-offline/README.md [hunk 4]
```diff
@@ -166,6 +166,28 @@ const options = {
...
+? What is the root of your web app (i.e. which directory do you deploy)? public/
```
Comment (laurieontech) [inline packages/gatsby-plugin-offline/README.md]: In order to have a public folder you need to have run `gatsby build` at least once. Would be good to note that's necessary for this setup.

File: packages/gatsby-plugin-offline/README.md [hunk 5]
```diff
@@ -166,6 +166,28 @@ const options = {
...
+? Which file types would you like to precache?
```
Comment (laurieontech) [inline packages/gatsby-plugin-offline/README.md]: Is the answer here none? Or all?

File: packages/gatsby-plugin-offline/README.md [hunk 6]
```diff
@@ -166,6 +166,28 @@ const options = {
...
+```
+
+Copy the content from `workbox-config.js` into your `gatsby-plugin-offline` options.
+
+For instance, the following configuration does the trick for a full offline website
+
+```
+  "globDirectory": "public/",
+  "globPatterns": [
+    "**/*.{html,js,json,png,jpg,webmanifest,woff,woff2,ttf,eot,css,mjs}"
```
Comment (laurieontech) [inline packages/gatsby-plugin-offline/README.md]: These globPatterns are different than those generated following the instructions above. Is this intentional?

**Comment by KyleAMathews** [comment]
@fheslouin hi do you think you'll have some time to finish this up?

**Comment by marcysutton** [comment]
Hi @fheslouin, checking in on this PR–are you able to make the requested changes?

**Comment by LekoArts** [comment]
I'll close this as it got stale. If anyone wants to pick this up, feel free to send in a new PR. Thanks for the initial PR though!

**Pull request closed without merging**